src/Xamarin.Android.Build.Tasks/Tasks/NdkUtils.cs

```diff
@@ -153,7 +153,9 @@ public static string GetNdkTool (string androidNdkPath, AndroidTargetArch arch,
 				toolName = $"{toolName}{extension}";
 			else
 				toolName = $"{toolchainPrefix}-{toolName}{extension}";
-			string toolPath  = Path.Combine (toolchainDir, "bin", toolName);
+
+			string toolExe  = MonoAndroidHelper.GetExecutablePath (toolchainDir, toolName);
```

Comment:
This change doesn't seem semantically equivalent: the previous version looks at `{toolchainDir}/bin/{toolName}`, while the new version looks at `{toolchainDir}/{toolName}%EXT%`.  The `bin` directory has been removed.

That said, the [Windows build succeeded](
https://devdiv.visualstudio.com/0bdbc590-a062-4c3f-b0f6-9383f67865ee/_build/results?buildId=2370254)...but I don't think it actually ran any unit tests.

Let's say I'm "concerned" and would like verification of the correct path? :-)